x/config/chooser_test.go

```diff
@@ -267,6 +266,42 @@ func TestChooserConfigurator(t *testing.T) {
 			},
 		},
 		{
+			desc: "HTTP",
+			given: whitespace.Expand(`
```

Comment:
This test had to be in the _test package to avoid a circular dependency, so the expand function wasn’t accessible for both this and the configurator test. The configurator test was too difficult to move into the _test package. May as well.